### DISCUSSION TITLE
Keeping the "<small>" tag in the TOC

### DISCUSSION THREAD

**Discussion by mgriff88** [squidfunk/mkdocs-material#7785] [Q&A]
Hi, I noticed on your Changelog page, you are able to keep the `<small>` tag inside the TOC. Can you share how this is done? Right now `striptags` takes it out. If possible, I'd like most tags to be stripped, but I do like how this one displays, so I'm wondering how you managed it. Thanks!
https://squidfunk.github.io/mkdocs-material/changelog/
![image](https://github.com/user-attachments/assets/19aec154-4dfa-4756-8c27-07a631499623)

**Comment by squidfunk** [comment]
Yes, you'll need to use [the typeset plugin](https://squidfunk.github.io/mkdocs-material/plugins/typeset/), which is [currently reserved to Sponsors](https://github.com/sponsors/squidfunk). With [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/), you just need to add the following:

``` yaml
plugins:
  - typeset
```

**Reply by mgriff88** [reply]
Got it, thank you, @squidfunk
